src/main/java/training/subsystems/Elbow/ElbowSubsystem.java

```diff
@@ -0,0 +1,76 @@
+package training.subsystems.Elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.subsystems.Elbow.Constants.*;
+
+public class ElbowSubsystem extends GBSubsystem {
+    private static ElbowSubsystem instance;
+
+    private final CANSparkMax motor = new CANSparkMax(ELBOW_ID, CANSparkLowLevel.MotorType.kBrushless);
+    private final SparkPIDController motorPIDF = motor.getPIDController();
+    private Rotation2d angle = DEFAULT_POSITION_ELBOW;
+
+    private ElbowSubsystem() {
+        motorPIDF.setP(ELBOW_PID_CONTROLLER.getP());
+        motorPIDF.setD(ELBOW_PID_CONTROLLER.getD());
+        motorPIDF.setI(ELBOW_PID_CONTROLLER.getI());
+    }
+
+    public static ElbowSubsystem getInstance() {
+        if (instance == null) {
+            instance = new ElbowSubsystem();
+        }
+        return instance;
+    }
```

Comment:
i think it should be above all like:
private Arm instance;
getInstance()

other variables
contructur
functions

Reply:
I fixed to this form but the default formatted hates this